src/main/scala/org/broadinstitute/dsde/workbench/sam/dataAccess/LdapRegistrationDAO.scala

```diff
@@ -99,15 +99,7 @@ class LdapRegistrationDAO(
   }
 
   override def isEnabled(subject: WorkbenchSubject, samRequestContext: SamRequestContext): IO[Boolean] =
-    for {
-      entry <- executeLdap(IO(ldapConnectionPool.getEntry(directoryConfig.enabledUsersGroupDn, Attr.member)), "isEnabled", samRequestContext)
-    } yield {
-      val result = for {
-        e <- Option(entry)
-        members <- Option(e.getAttributeValues(Attr.member))
-      } yield members.contains(subjectDn(subject))
-      result.getOrElse(false)
-    }
+    IO.raiseError(new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.InternalServerError, s"LdapRegistrationDAO is deprecated")))
```

Comment:
not sure if this is the best way to deprecate this call -- let me know if you have ideas!

Reply:
I'd be tempted to add a `@deprecated` annotation to this class if it is indeed deprecated